### PR TITLE
[accel brake map]update readme

### PR DESCRIPTION
**Pull request by tkimura4**
## Description

- Add description about diagnostics published by accel_brake_map_calibrator
etc

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by tkimura4** [comment]
FYI @taikitanaka3 
https://github.com/autowarefoundation/autoware.universe/pull/80

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 1]
```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
 
 The value of each cell in the map is gray at first, and it changes from blue to red as the number of valid data in the cell accumulates. It is preferable to continue the calibration until each cell of the map becomes close to red. In particular, the performance near the stop depends strongly on the velocity of 0 ~ 6m/s range and the pedal value of +0.2 ~ -0.4, range so it is desirable to focus on those areas.
 
-### Check the recommended notifications for map updates
+### Diagnostics
+
+`accel brake map_calibrator` publishes diagnostics depends on calibration status.
```
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:46]: ???
```suggestion
`accel brake map_calibrator` publishes diagnostics depending on calibration status.
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 2]
```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
 
 The value of each cell in the map is gray at first, and it changes from blue to red as the number of valid data in the cell accumulates. It is preferable to continue the calibration until each cell of the map becomes close to red. In particular, the performance near the stop depends strongly on the velocity of 0 ~ 6m/s range and the pedal value of +0.2 ~ -0.4, range so it is desirable to focus on those areas.
 
-### Check the recommended notifications for map updates
+### Diagnostics
+
+`accel brake map_calibrator` publishes diagnostics depending on calibration status.
```
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:46]: The `accel brake map_calibrator` publishes diagnostics message depending on the calibration status.

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 3]
```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
...
+When the diagnostics type is `WARN`, the accuracy of current accel/brake map may be low and the accuracy improvement is strongly expected by updating the maps.
```
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:47]:
```suggestion
Diagnostic type `WARN` indicates that the current accel/brake map is estimated to be inaccurate. In this situation, it is strongly recommended to perform a re-calibration of the accel/brake map.
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 4]
```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
...
+
+| Status                  | Diagnostics Type | Diagnostics message                        | Description                                         |
+| ----------------------- | ---------------- | ------------------------------------------ | --------------------------------------------------- |
+| No calibration required | `OK`             | "OK"                                       |                                                     |
+| Calibration Required    | `WARN`           | "Accel/brake map Calibration is required." | The accuracy of current accel/brake map may be low. |
+
+This diagnostics status can be also checked on Ros topic.
```
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:54]:
```suggestion
This diagnostics status can be also checked on the following ROS topic.
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md [hunk 5]
```diff
@@ -41,19 +41,29 @@ Note: You don't need to worry about whether the current state is red or green du
...
 
 ```sh
 ros2 topic echo /accel_brake_map_calibrator/output/update_suggest
 ```
 
-(It is desirable to monitor this topic while driving)
+When the diagnostics type is `WARN`, `True` is published on this topic.
```
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:60]:
```suggestion
When the diagnostics type is `WARN`, `True` is published on this topic and the update of the accel/brake map is suggested.
```

File: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md (hunk 2 above)
Comment (TakaHoribe) [inline vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md:46]:
```suggestion
The `accel brake map_calibrator` publishes diagnostics message depending on the calibration status.
```

**Review by TakaHoribe — APPROVED** [review]

**Pull request merged**